LondonBritishLibrary/orient/BLorient547.xml

```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.6">
+<locus from="140r"/>
+<title xml:lang="en" type="complete" ref="LIT1098Anapho">Anaphora of Basil</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.7">
+<locus from="152r"/>
+<title xml:lang="en" type="complete" ref="LIT2040Naakkw">Anaphora of Gregory, the brother of Basil</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.8">
+<locus from="159v"/>
+<title xml:lang="en" type="complete" ref="LIT1347Epipha">Anaphora of Epiphanius</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.9">
+<locus from="167r"/>
+<title xml:lang="en" type="complete" ref="LIT3180Anapho">Anaphora of Dioscorus</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.10">
+<locus from="171r"/>
+<title xml:lang="en" type="complete" ref="LIT1569habeka">Anaphora of Cyril</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.11">
+<locus from="179r"/>
+<title xml:lang="en" type="complete" ref="LIT3120Anapho">Anaphora of John Chrysostom</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.12">
+<locus from="186r"/>
+<title xml:lang="en" type="complete" ref="LIT2404Tanseu">Anaphora of Jacob of Sarug</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+
+                  </msContents>
+<physDesc>
+
+  <handDesc>
+  <handNote script="Ethiopic" xml:id="h2">
+  <desc>The manuscript is written in a good hand, towards the end of <date>18th century</date>.</desc>
+  <date notBefore="1750" notAfter="1800"/>
+  </handNote>
+  </handDesc>
+
+  <additions>
+  <list>
+
+  <item xml:id="a1">
+  <locus target="#4r"/>
+  <desc type="OwnershipNote">The manuscript was written for a woman called <persName ref="PRS13410Walatta">Walatta Takla Hāymānot</persName>. Her
+  name is also mentioned on <locus target="#13v">f. 13v</locus> and <locus target="#14r">14r</locus>, with that of her father <persName ref="PRS13409Hensa">Ḥǝnḍā Śǝllāse</persName> and
+  her brother <persName ref="PRS13412Haile">Ḫayla Takla Hāymānot</persName>.</desc>
+  </item>
+
+  <item xml:id="a2">
+  <desc type="MixedNote">The name of the king <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName> often occurs (e. g. on <locus target="#12v">f. 12v</locus>) and
+  the sitting ʾabuna was <persName ref="PRS10450YosabI">ʾIyosāb</persName>. (<locus target="#45r">f. 45r</locus>, <locus target="#65r">65r</locus>). The name of the
+  patriarch of Alexandria is left blank.</desc>
+  </item>
+
+  <item xml:id="a3">
+  <locus target="#4r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ቅዳሴ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+  </item>
+
+  <item xml:id="e1">
+  <desc type="MixedNote">The name <persName ref="PRS13414Walatta_">Walatta Giyorgis</persName> occurs
```

Comment:
I do not think that the type is needed here.

Reply:
sure